test/setup/iam.tf

```diff
@@ -55,25 +69,41 @@ resource "google_service_account" "gke_sa_2" {
 }
 
 resource "google_project_iam_member" "int_test_1" {
-  count = length(local.int_required_roles)
+  count = length(local.int_required_project_roles)
 
   project = module.gke-project-1.project_id
-  role    = local.int_required_roles[count.index]
+  role    = local.int_required_project_roles[count.index]
   member  = "serviceAccount:${google_service_account.int_test.email}"
 }
 
 resource "google_project_iam_member" "int_test_2" {
-  count = length(local.int_required_roles)
+  count = length(local.int_required_project_roles)
 
   project = module.gke-project-2.project_id
-  role    = local.int_required_roles[count.index]
+  role    = local.int_required_project_roles[count.index]
   member  = "serviceAccount:${google_service_account.int_test.email}"
 }
 
+resource "google_folder_iam_member" "int_test_folder" {
+  count = length(local.int_required_folder_roles)
+
+  folder = google_folder.ci_gke_folder.name
+  role   = local.int_required_folder_roles[count.index]
+  member = "serviceAccount:${google_service_account.int_test.email}"
+}
+
 resource "google_service_account_key" "int_test" {
   service_account_id = google_service_account.int_test.id
 }
 
+
+resource "google_billing_account_iam_member" "int_billing_user" {
```

Comment:
This role shouldn't be necessary since all projects are created in this configuration.

Reply:
done